app/services/acceptable_verified_credential_adoption_service.rb

```diff
@@ -8,15 +8,15 @@
 class AcceptableVerifiedCredentialAdoptionService
   attr_accessor :user
 
-  REACTIVATION_TEMPLATE = Settings.vanotify.services.va_gov.template_id.login_reactivation_email
+  REACTIVATION_TEMPLATE_ID = Settings.vanotify.services.va_gov.template_id.login_reactivation_email
```

Comment:
is there a reason to have this as a setting instead of hard coded here? Since it's a constant it will still require a deploy to change this

Reply:
Oh I read the PR description, yes, this appears to be the intention.

Reply:
Initially it was because we anticipated having to send out multiple emails over time via this service.  Now that we've pivoted to using GovDelivery for future sends, this isn't necessary anymore.  

I can remove it and use a hardcoded version or we can keep it as is and it'll be removed once we fully transition to the GovDelivery service in August.

Reply:
I removed it and went hardcoded instead, per your suggestion :)